### GeoPolygon message
#
# A specification of a polygon with (latitude, longitude, altitude) points
# and where the first and last points are assumed to be connected if not equal.
geographic_msgs/GeoPoint[]  points   # [degrees, degrees, meters]
